dist/bin/dotr

```diff
@@ -65,7 +74,8 @@ while [[ $# -gt 0 ]]; do
       shift
       ;;
     -J*)
-      addJvmOptions "-${1:2}"
+      addJvmOptions "${1:2}"
+      addRawJvmOptions "${1}"
```

Comment:
Maybe rename to something like `addJvmOptionsRun/addJvmOptionsRepl` or `addJvmOptions/addDotcOptions`, which is more clear?